weather_mv/loader_pipeline/bq.py

```diff
@@ -35,6 +35,9 @@
 DATA_IMPORT_TIME_COLUMN = 'data_import_time'
 DATA_URI_COLUMN = 'data_uri'
 DATA_FIRST_STEP = 'data_first_step'
+S2_LOCATION = 's2_location'  # st_geog_point
+MIN_LATITUDE = -90
+MAX_LATITUDE = 90
```

Comment:
Optional nit: it might be appropriate to store this as a tuple, since these two values are address the same concern. WDYT?